api/src/clojure/org/akvo/flow_api/component/akvo_flow_server_config.clj

```diff
@@ -37,7 +37,7 @@
   component/Lifecycle
   (start [this]
     (assoc this
-      :flow-config (atom {"akvoflowsandbox" "something"})))
+      :flow-config (atom {:instances {"akvoflowsandbox" "something"}})))
```

Comment:
i don't see any other `:instances` reference in this PR, so I derive that we missed this thing previously, don't we?

Reply:
👍 thanks for the clarifications!